pkg/auth/authinfo/impl.go

```diff
@@ -49,3 +58,36 @@ func (ai *AuthInfo) checkGroup(group string) bool {
 	_, ok := ai.groups[group]
 	return ok
 }
+
+func (ai *AuthInfo) write(writer io.Writer,
+	prefix, space, postfix string) error {
+	if ai == nil {
+		fmt.Fprintf(writer, "%sNo authentication information%s\n",
+			prefix, postfix)
+	} else {
```

Comment:
remove the else and just return nil on the if branch...

Reply:
Done.